ansible/roles/tldraw-server/templates/configmap.yml.j2

```diff
@@ -6,16 +6,13 @@ metadata:
   labels:
     app: tldraw-server
 data:
-  API_HOST: "http://api-svc:3030"
-  WS_PATH_PREFIX: "/tldraw-server"
-  WS_PORT: "3345"
-  FEATURE_PROMETHEUS_METRICS_ENABLED: "true"
+  AUTHORIZATION_API_HOST: "{{ API_HOST_URI }}"
+  LOGGER_LOG_LEVEL: "{{ TLDRAW_LOG_LEVEL }}"
+  METRICS_COLLECT_DEFAULT: "{{ TLDRAW_METRICS_COLLECT_DEFAULT }}"
   REDIS_CLUSTER_ENABLED: "true"
   REDIS_SENTINEL_SERVICE_NAME: "valkey-headless.{{ NAMESPACE }}.svc.cluster.local"
-  TLDRAW__WEBSOCKET_URL: "wss://{{ DOMAIN }}/tldraw-server"
-  TLDRAW__ASSETS_ENABLED: "true"
-  TLDRAW__ASSETS_MAX_SIZE_BYTES: "10485760"
-  TLDRAW__ASSETS_ALLOWED_MIME_TYPES_LIST: "image/png,image/jpeg,image/gif,image/svg+xml"
-  FEATURE_TLDRAW_ENABLED: "true"
-  NEST_LOG_LEVEL: "info"
-  EXIT_ON_ERROR: "true"
```

Comment:
no more exit on error?

Reply:
Is default true in code

Reply:
After all i personally think it's a very bad Idea to exit on error.
Specaly on a multi user and board POD.
Why i think so is that if for one user an error happened all other users with other boards are even face the consequences of the restart.